defaults/main.yml

```diff
@@ -12,6 +12,8 @@ kubernetes_packages:
 kubernetes_role: master
 
 kubernetes_kubelet_extra_args: ""
+# Any additional options for kubeadm init
+kubeadm_init_opts: ""
```

Comment:
Can you remove the comment above, and change the variable to:

`kubernetes_kubeadm_init_extra_opts: ""`

Then also add mention of it in the README.md file along with the other vars documented there (under the entry for `kubernetes_kubelet_extra_args`).

Thanks!